google/resource_compute_disk_test.go

```diff
@@ -292,7 +292,7 @@ func testAccComputeDisk_deleteDetach(instanceName, diskName string) string {
 	return fmt.Sprintf(`
 resource "google_compute_disk" "foo" {
 	name = "%s"
-	image = "debian-8"
+	image = "debian-8-jessie-v20170523"
```

Comment:
It seems that before https://github.com/hashicorp/terraform/pull/14573 the `google_compute_disk` resource didn't READ much back from the API. I suppose the API allows `debian-8` to initialize as a convince for the user, but the API seems to give it a more concrete value in the backend. 14573 introduced reading these values back, thus we see the diff shown in the test failures. 

I believe the new behavior is the correct behavior, and we should discourage users from simply using the `debian-8` shorthand going forward.